lib/src/util/vulnerabilitySeverityLevels.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+/**
+ * All Possible Vulnerability (CVSS) Severity Levels.
+ */
+export const allVulnerabilitySeverityLevels = [
+  'none',
+  'low',
+  'medium',
+  'high',
+  'critical'
+] as const;
+
+type SingleDigit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9';
+
+export type VulnerabilityScore = `${SingleDigit}.${SingleDigit}` | '10.0';
+
+/**
+ * The type containing only the possible vulnerability CVSS severity level values.
+ */
+export type VulnerabilitySeverityLevel =
+  (typeof allVulnerabilitySeverityLevels)[number];
+
+/**
+ * Lists the ranges of Vulnerability (CVSS) scores and their corresponding severity level.
+ */
+export const vulnerabilitySeverityAndScoreRangeList = Object.freeze([
```

Comment:
Per https://nvd.nist.gov/vuln-metrics/cvss the mapping is actually different/simpler for CVSS 2 than for CVSS 3 and 4.  What requirements do we have related to worrying about that or not?

Reply:
> Per https://nvd.nist.gov/vuln-metrics/cvss the mapping is actually different/simpler for CVSS 2 than for CVSS 3 and 4. What requirements do we have related to worrying about that or not?

We're only required to support CVSS 3 as of this moment from my discussions with the other teams.